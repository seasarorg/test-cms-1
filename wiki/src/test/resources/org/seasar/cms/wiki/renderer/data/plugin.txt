|&color( blue , , fff){aaa};|

&size(9){&color(#000,#bdf){ページ作成方針};};
&size(9){複数のインライン要素が%%子要素%%に};

&calc(+,&calc(-,5,3),1);~
&calc(+,&calc(-,5,&calc(+,1,2)),1);

&color(blue){1行に二つのプラグイン}; &size(12){1行に二つのプラグイン};

#contents

#clear

