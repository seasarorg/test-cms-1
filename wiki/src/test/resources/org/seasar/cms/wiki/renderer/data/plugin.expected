<table><tbody><tr><td>
<span style="color:blue; background-color:">aaa</span>

</td>
</tr></tbody>
</table>
<p><span style="font-size:9px"><span style="color:#000; background-color:#bdf">ページ作成方針</span>
</span>
<span style="font-size:9px">複数のインライン要素が<del>子要素</del>に</span>
</p><p><br/></p><p><span style="color:blue">1行に二つのプラグイン</span>
 <span style="font-size:12px">1行に二つのプラグイン</span>
</p><div class="contents"></div>

<div style="clear:both;padding-bottom:1px;"></div>
